objects/Application/myOperations Patch Management.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-16"?>
+<AppMgmtDigest xmlns="http://schemas.microsoft.com/SystemCenterConfigurationManager/2009/AppMgmtDigest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <Application AuthoringScopeId="ScopeId_1A82C462-52AF-4BB7-9B3E-32D6A9C34589" LogicalName="Application_89eebb7e-b2c0-4edb-81cc-50db2aa519ca" Version="1">
+    <DisplayInfo DefaultLanguage="en-US">
+      <Info Language="en-US">
+        <Title>myOperations Patch Management</Title>
+        <Description />
+        <Publisher>Axians Cloud und IT Automation GmbH</Publisher>
+        <Version>3.32.0</Version>
+      </Info>
+    </DisplayInfo>
+    <DeploymentTypes>
+      <DeploymentType AuthoringScopeId="ScopeId_1A82C462-52AF-4BB7-9B3E-32D6A9C34589" LogicalName="DeploymentType_0a81f2eb-65e8-496b-83fc-96d0388d6d69" Version="1" />
+    </DeploymentTypes>
+    <Title ResourceId="Res_1665628948">myOperations Patch Management</Title>
+    <Description ResourceId="Res_1447935118" />
+    <Publisher ResourceId="Res_1043394885">Axians Cloud und IT Automation GmbH</Publisher>
+    <SoftwareVersion ResourceId="Res_860071511">3.32.0</SoftwareVersion>
+    <CustomId ResourceId="Res_1502952390" />
+    <Owners>
+      <User Qualifier="LogonName" Id="mkoe" />
+    </Owners>
+    <Contacts>
+      <User Qualifier="LogonName" Id="mkoe" />
+    </Contacts>
+  </Application>
+  <DeploymentType AuthoringScopeId="ScopeId_1A82C462-52AF-4BB7-9B3E-32D6A9C34589" LogicalName="DeploymentType_0a81f2eb-65e8-496b-83fc-96d0388d6d69" Version="1">
+    <Title ResourceId="Res_2110917655">myOperations Portal - Windows Installer (*.msi file)</Title>
+    <DeploymentTechnology>GLOBAL/MSIDeploymentTechnology</DeploymentTechnology>
+    <Technology>MSI</Technology>
+    <Hosting>Native</Hosting>
+    <Installer Technology="MSI">
+      <ExecutionContext>System</ExecutionContext>
+      <Contents>
+        <Content ContentId="Content_16c90ea8-95ca-49ca-900e-551d1766c663" Version="1">
+          <File Name="myoperations Portal PM 3.32.0.201.msi" Size="287998242" />
+          <Location>\\T00-MYOPS-SCCM\Applications\</Location>

```

Comment:
FYI anyone using this app will need to update with their own location (and have their own copy of this .msi file).